src/scenes/home/index.jsx

```diff
@@ -13,9 +13,15 @@ import {
 import { Alert } from '@material-ui/lab'
 import { Delete } from '@material-ui/icons'
 
+import React from 'react'
+import Modal from 'react-modal'
```

Comment:
There is actually a Modal component in material-ui, so pulling in a new library is kind of unnecessary here. However, react-modal is something you're more likely to use IRL than mat-ui, so I'm ok with it being used here for some practice.